imap/http_dav.c

```diff
@@ -6409,6 +6421,9 @@ EXPORTED int meth_propfind(struct transaction_t *txn, void *params)
     xml_partial_response(txn, fctx.root->doc, NULL /* end */, 0, &fctx.xmlbuf);
     xmlBufferFree(fctx.xmlbuf);
 
+    // might have made a change!
+    sync_checkpoint(txn->conn->pin);
+
```

Comment:
@brong maybe its just too early for me, but what could PROPFIND actually change.  Its a safe (read-only) method

Reply:
It can create mailboxes if they don't exist yet!